Acceptance-Tests.md

```diff
@@ -216,6 +216,35 @@ const CONSOLE_ERRORS_TO_FIX = [
 ];
 ```
 
+### Screenshots testing functionality in Acceptance Tests
+
+Acceptance Tests have a function called `expectScreenshotToMatch` in `puppeteer-utils.ts` to take screenshots of the UI during the acceptance tests and compare them to the existing screenshots in the codebase, which can help debugging test failures as it provides more information beside the error message. To use this functionality, call the function `expectScreenshotToMatch`, which takes in a string as the name of the screenshot, and the absolute path of the directory of the specs file to locate where the folder of the screenshots will be. For instance, to create a screenshot after calling the function `loggedOutUser.clickTeachButtonInAboutMenuOnNavbar` in `logged-out-user/click-all-buttons-on-navbar.spec.ts`, call the function `expectScreenshotToMatch` with the user type `loggedOutUser` to identify which user's browser should be screenshotted, `teachPage`, a name for the screenshots that describes what the page is, and the variable `__dirname`. 
```

Comment:
which can help debugging --> which can help with debugging

I suggest adding a paragraph break before "TO use this functionality"